packages/sdk/src/swaps/smart-router/routes/mynt-fixed-rate-route.ts

```diff
@@ -0,0 +1,140 @@
+import { Contract, constants, providers } from 'ethers';
+
+import {
+  SupportedTokens,
+  getProtocolContract,
+  getTokenContract,
+} from '@sovryn/contracts';
+import { ChainId, numberToChainId } from '@sovryn/ethers-provider';
+
+import { SovrynErrorCode, makeError } from '../../../errors/errors';
+import {
+  canSwapPair,
+  makeApproveRequest,
+  hasEnoughAllowance,
+} from '../../../internal/utils';
+import { SwapPairs, SwapRouteFunction } from '../types';
+
+export const myntFixedRateRoute: SwapRouteFunction = (
+  provider: providers.Provider,
+) => {
+  let pairCache: SwapPairs;
+  let chainId: ChainId;
+  let fixedRateMyntContract: Contract;
+  let protocolContract: Contract;
+
+  const getChainId = async () => {
+    if (!chainId) {
+      chainId = numberToChainId((await provider.getNetwork()).chainId);
+    }
+    return chainId;
+  };
+
+  const getFixedRateMyntContract = async () => {
+    const chainId = await getChainId();
+    const { address, abi } = await getProtocolContract(
+      'fixedRateMynt',
+      chainId,
+    );
+    return new Contract(address, abi, provider);
+  };
+
+  const getConverterContract = async () => {
+    if (!fixedRateMyntContract) {
+      fixedRateMyntContract = await getFixedRateMyntContract();
+    }
+    return fixedRateMyntContract;
+  };
+
+  const getSwapQuoteContract = async () => {
+    if (!protocolContract) {
+      const chainId = await getChainId();
+      const { address, abi } = await getProtocolContract('protocol', chainId);
+      protocolContract = new Contract(address, abi, provider);
+    }
+    return protocolContract;
+  };
+
+  return {
+    name: 'MyntFixedRate',
+    pairs: async () => {
+      if (!pairCache) {
+        const chainId = await getChainId();
+
+        const swapTokens = [SupportedTokens.sov, SupportedTokens.mynt];
+
+        const contracts = await Promise.all(
+          swapTokens.map(token => getTokenContract(token, chainId)),
+        );
+
+        const addresses = contracts.map(contract =>
+          contract.address.toLowerCase(),
+        );
+
+        const pairs = new Map<string, string[]>();
+
+        for (const address of addresses) {
+          const pair = addresses.filter(a => a !== address);
+          pairs.set(address, pair);
+        }
+
+        pairCache = pairs;
+      }
+
+      return pairCache;
+    },
+    quote: async (entry, destination, amount) => {
+      return (await getSwapQuoteContract())
+        .getSwapExpectedReturn(
+          entry.toLowerCase(),
+          destination.toLowerCase(),
+          amount,
+        )
```

Comment:
Use fixedMynt contract to get quote:

```suggestion
      return (await getConverterContract())
        .convertAmount(
          amount
        )
```

Rate is actually also fixed to `0.004723550439442834` according to the PRD, so rpc call could be bypassed and amount can be calculated in js too.

ABI file also has `convertMax` function - which suggest that max convertable MYNT should be limited too